packages/admin-portal/src/sagas/approvals/approvals.ts

```diff
@@ -1,35 +1,32 @@
-import { approvalsLoading, approvalsReceiveData, approvalsRequestDataFailure } from '@/actions/approvals'
+import { fetchApprovalListSuccess, fetchApprovalListFailed } from '@/actions/approvals'
 import { put, fork, takeLatest, all, call } from '@redux-saga/core/effects'
 import ActionTypes from '@/constants/action-types'
-import { errorThrownServer } from '@/actions/error'
-import errorMessages from '@/constants/error-messages'
 import { Action } from '@/types/core'
-import { logger } from '@reapit/utils'
+import { logger, extractNetworkErrString } from '@reapit/utils'
 import { fetchApprovalsList } from '@/services/approvals'
+import { notification } from '@reapit/elements'
 
 export const approvalsDataFetch = function*({ data: page }) {
-  yield put(approvalsLoading(true))
-
   try {
     const response = yield call(fetchApprovalsList, { pageNumber: page })
     if (response) {
-      yield put(approvalsReceiveData(response))
+      yield put(fetchApprovalListSuccess(response))
     } else {
-      yield put(approvalsRequestDataFailure())
+      yield put(fetchApprovalListFailed())
     }
   } catch (err) {
     logger(err)
-    yield put(
-      errorThrownServer({
-        type: 'SERVER',
-        message: errorMessages.DEFAULT_SERVER_ERROR,
-      }),
-    )
+    const networkErrorString = extractNetworkErrString(err)
+    yield call(notification.error, {
```

Comment:
I think no need to use yield here, because notification can pop out withou yield

Reply:
I think that is the standard way to call side effect function in sagas, and test it without mocking notification object